.ci/windows-steps.yaml

```diff
@@ -118,6 +116,60 @@ steps:
     replaceExistingArchive: true
   displayName: 'Build artifacts'
 
+# Build MSI installer.
+- powershell: |
+    # Pull the documentation for the installer.
+    try {
+      $url = "http://ci.mlpack.org/job/mlpack%20-%20doxygen%20build/lastSuccessfulBuild/artifact/build/doc/html/*zip*/html.zip"
+      (new-object net.webclient).DownloadFile($url, 'dist\win-installer\jenkinsdoc.zip')
+    }
+    catch {
+      Write-Output "Unable to download precompiled Doxygen documentation from Jenkins!"
+    }
+    try {
+      (Add-Type -AssemblyName System.IO.Compression.FileSystem);
+      [System.IO.Compression.ZipFile]::ExtractToDirectory('dist\win-installer\jenkinsdoc.zip', 'dist\win-installer\mlpack-win-installer\Sources\doc')
+    }
+    catch{Write-Output "Unable to add doc to installer, skipping!"}
+    # Preparing installer staging.
+    mkdir dist\win-installer\mlpack-win-installer\Sources\lib
+    cp build\Release\*.lib dist\win-installer\mlpack-win-installer\Sources\lib\
+    cp build\Release\*.exp dist\win-installer\mlpack-win-installer\Sources\lib\
+    cp build\Release\*.dll dist\win-installer\mlpack-win-installer\Sources\
+    cp build\Release\*.exe dist\win-installer\mlpack-win-installer\Sources\
+    cp $(Agent.ToolsDirectory)\OpenBLAS.0.2.14.1\lib\native\bin\x64\*.dll dist\win-installer\mlpack-win-installer\Sources\
+    cp build\include\mlpack dist\win-installer\mlpack-win-installer\Sources -recurse
+    cp doc\examples dist\win-installer\mlpack-win-installer\Sources -recurse
+    cp src\mlpack\tests\data\german.csv dist\win-installer\mlpack-win-installer\Sources\examples\sample-ml-app\sample-ml-app\data\
+    # Check current git version or mlpack version.
+    $ver = (Get-Content "src\mlpack\core\util\version.hpp" | where {$_ -like "*MLPACK_VERSION*"});
+    $env:MLPACK_VERSION += $ver[0].substring($ver[0].length - 1, 1) + '.';
+    $env:MLPACK_VERSION += $ver[1].substring($ver[1].length - 1, 1) + '.';
+    $env:MLPACK_VERSION += $ver[2].substring($ver[2].length - 1, 1);
+
+    if (Test-Path "src/mlpack/core/util/gitversion.hpp")
+    {
+      $ver = (Get-Content "src/mlpack/core/util/gitversion.hpp");
+      $env:INSTALL_VERSION = $ver.Split('"')[1].Split(' ')[1];
+    }
+    else
+    {
+      $env:INSTALL_VERSION = $env:MLPACK_VERSION;
+    }
+
+    # Build the MSI installer.
+    cd dist\win-installer\mlpack-win-installer
+    dir
+    cd Sources
+    dir
```

Comment:
Do we like to keep the `dir` here for future debug sessions, or can we remove them?

Reply:
Good point, thanks!  Removed in 36abf9577.